snow/engine/snowman/syncer/state_syncer_test.go

```diff
@@ -1262,7 +1262,7 @@ func TestStateSyncIsStoppedIfEnoughVotesAreCastedWithNoClearMajority(t *testing.
 				context.Background(),
 				voterID,
 				reqID,
-				set.Of(ids.ID{'u', 'n', 'k', 'n', 'o', 'w', 'n', 'I', 'D'}),
+				set.Of(ids.GenerateTestID()),
```

Comment:
Was it ever useful in debugging to have these kinds of "labelled" ids? If so, you could add `ids.GenerateLabelledTestID(tb testing.TB, lbl string)` that then calls `tb.Logf("test ID %q: %v", lbl, id)` before returning the value.

The only problem I see with this would be introducing the `testing` package into prod code.

Reply:
This is completely optional and dependent on the answer to my opening question.